public/index.html

```diff
@@ -9,6 +9,8 @@
       name="description"
       content="Web site created using create-react-app"
     />
```

Comment:
```suggestion
    />
    <link rel="preconnect"  href="https://fonts.googleapis.com" />
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet">
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

for better performance

Reply:
Sure will update this. Thanks.